metrics/config.go

```diff
@@ -40,32 +40,32 @@ const (
 	// The following keys are used to configure metrics reporting.
 	// See https://github.com/knative/serving/blob/master/config/config-observability.yaml
 	// for details.
-	AllowStackdriverCustomMetricsKey = "metrics.allow-stackdriver-custom-metrics"
+	allowStackdriverCustomMetricsKey = "metrics.allow-stackdriver-custom-metrics"
 	BackendDestinationKey            = "metrics.backend-destination"
-	ReportingPeriodKey               = "metrics.reporting-period-seconds"
+	reportingPeriodKey               = "metrics.reporting-period-seconds"
 	// Stackdriver client configuration keys
```

Comment:
Are we fine w/ mixing private/public in one "block" ?

or should the be "ordered" (private/public and alphabetically?)

Reply:
:shrug: I can do that.

Reply:
ok sorted and reordered and split enum alone.